cypress.config.js

```diff
@@ -1,18 +1,23 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 const { defineConfig } = require("cypress");
+const plugins = require("./cypress/plugins");
 
 module.exports = defineConfig({
   projectId: "4trcdv",
   defaultCommandTimeout: 10000,
   pageLoadTimeout: 10000,
   requestTimeout: 30000,
-  testFiles: "@(apps|components)/**/*.test.@(ts|tsx)",
-  integrationFolder: "./src",
   retries: {
     runMode: 3,
     openMode: 0
   },
   e2e: {
-    baseUrl: "http://localhost:57021"
+    supportFile: "cypress/support/index.ts",
+    specPattern: "./src/@(apps|components)/**/*.test.@(ts|tsx)",
+    baseUrl: "http://localhost:57021",
+    testIsolation: false,
```

Comment:
Good workaround for now. But I think that our tests should be updated in the future so that they are isolated